hugo/layouts/partials/card.html

```diff
@@ -0,0 +1,11 @@
+<article class="cards__element cards__element_category">
```

Comment:
since it is said that
> _Important! A double hyphen inside a comment (--) is interpreted as part of the comment, which causes an error during document validation. HTML5 Specification._

I'd prefer to use the "main" spec, and to use a single underscore `_` for modifiers